src/tasks/dimensionality_reduction/metrics/normalized_mutual_information/config.vsh.yaml

```
@@ -0,0 +1,53 @@
+__merge__: ../../api/comp_metric.yaml
+
+functionality:
+  name: normalized_mutual_information
+  info:
+    metrics:
+      - name: normalized_mutual_information
+        label: Normalized Mutual Information
+        summary: Measure of the concordance between clustering obtained from the original and reduced-dimensional representations.
```

I think this will look better in figures

```suggestion
        label: NMI
        summary: Normalized Mutual Information (NMI) is a measure of the concordance between clustering obtained from the original and reduced-dimensional representations.